prometheus/prometheus.yml

```diff
@@ -2,42 +2,79 @@ global:
   scrape_interval: 15s
```

Comment:
lets not check in the changes from this file since they're specific to sei-testnet-2